e2e/test/scenarios/admin-2/settings.cy.spec.js

```diff
@@ -665,49 +665,6 @@ describe("scenarios > admin > license and billing", () => {
         .should("have.prop", "tagName", "A");
     });
 
-    it("should show the user store info for an self-hosted instance managed by the store", () => {
```

Comment:
is there a way to keep this test? this feature is nearly impossible to test manually, so this test is all the coverage we have on it I think